pkg/planner/core/internal/base/plan.go

```diff
@@ -108,9 +108,9 @@ func (p *Plan) SetTP(tp string) {
 	p.tp = tp
 }
 
-// SelectBlockOffset is to get the select block offset.
-func (p *Plan) SelectBlockOffset() int {
-	return p.blockOffset
+// QBOffset is to get the select block offset.
+func (p *Plan) QBOffset() int {
```

Comment:
May I ask why we call it `QB`? Why not just call it `QueryBlock`? It is more obvious and less confusing.

Reply:
Got it. Make sense. Thank you. I guess QueryBlockOffset is also short enough. I don't have a strong opinion about it. So feel free to change it.

Reply:
You are right. I have fixed it.